packages/Checkbox/src/Checkbox.story.tsx

```diff
@@ -18,7 +27,7 @@ class Control extends PureComponent {
 
   state = { checked: false };
 
-  onChange = e => {
+  onChange = (e: React.ChangeEvent<HTMLInputElement>) => {
```

Comment:
I think I just default to annotating params/returns for functions, rather than annotating the whole function as a single value. They should be essentially equivalent, outside of `ChangeEventHandler` mandating that the function doesn't have a return value.

Reply:
Wouldn't that make `ChangeEventHandler` the preferred way to type handlers like this?

Reply:
I don't think it matters. I would say that the provided handler would be the way to go for defining a prop, except that I'd prefer it have a return type of `unknown` instead of `void` (since afaik any return values will just be ignored). For something that's used internally, I don't see any real reason to prefer one pattern over the other.

Reply:
My point here I suppose is: should this be the pattern we use? Best to keep things uniform if possible.